### PR TITLE
Fixed for mariadb

### PR DESCRIPTION
**Pull request by slalomzacharyd**
There were 2 hardcoded service['mysql-server'] that were replaced with the variable, making this module compatible with mariadb on centos7.

### CODE REVIEW

File: manifests/server/installdb.pp [hunk 1]
```diff
@@ -20,7 +19,7 @@
       creates   => "${datadir}/mysql",
       logoutput => on_failure,
       path      => '/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:/usr/local/sbin',
-      require   => Package['mysql-server'],
+      require   => Package[$mysql::params::server_package_name],
```
Comment (igalic) [inline manifests/server/installdb.pp]: this should be overridable in mysql::server, we shouldn't get it from params

File: manifests/server/service.pp [hunk 2]
```diff
@@ -31,7 +31,7 @@
     name     => $mysql::server::service_name,
     enable   => $mysql::server::real_service_enabled,
     provider => $mysql::server::service_provider,
-    require  => Package['mysql-server'],
+    require  => Package[$mysql::params::server_package_name],
```
Comment (igalic) [inline manifests/server/service.pp]: same as above.

**Comment by slalomzacharyd** [comment]
Added the changes you requested.

**Comment by igalic** [comment]
could you please [squash](http://www.git-scm.com/book/en/Git-Tools-Rewriting-History#Squashing-Commits) these commits?

**Comment by slalomzacharyd** [comment]
I can.  But it's honestly much easier if you just squash on merge with the commit message that makes sense for you instead of my doing a rebase on a branch that will be thrown out after this.

**Comment by igalic** [comment]
okay! thank you @slalomzacharyd

rebased in #718

**Pull request closed without merging**